spec/models/dialog_field_text_box_spec.rb

```diff
@@ -43,6 +43,7 @@
 
       context "when show_refresh_button is false" do
         let(:show_refresh_button) { false }
+        let(:load_values_on_init) { true }
```

Comment:
Similar to the above specs that have multiple contexts for whether or not `load_values_on_init` is true/false, I think this context also needs those contexts since the logic is changing, instead of simply setting it `true` here just to make this one spec pass as it used to.

Essentially, we're missing the case where `show_refresh_button` is false and `load_values_on_init` is also false, since this is a thing that can actually happen now.

Reply:
This *is* as it used to.

But agreed, we need a new spec for the situation where both are false.

Reply:
Right, but I meant as opposed to *just* setting this value so that it works like it used to. Sorry, my wording was poor.